packages/ui-tree-browser/src/TreeBrowser/index.tsx

```diff
@@ -73,7 +73,11 @@ class TreeBrowser extends Component<TreeBrowserProps> {
     // @ts-expect-error ts-migrate(6133) FIXME: 'id' is declared but its value is never read.
     onCollectionClick: function (id, collection) {},
     // @ts-expect-error ts-migrate(6133) FIXME: 'collection' is declared but its value is never re... Remove this comment to see the full error message
-    onCollectionToggle: function (collection) {}
+    onCollectionToggle: function (collection) {},
+    // @ts-expect-error ts-migrate(6133) FIXME: 'foo' and 'bar' are declared but their values are never read.
+    sortOrder: function (obj1, obj2) {
+      return 0
+    }
```

Comment:
Do not add a default value if it does nothing, rather add null checks (e.g. with `?.`) where this is used.